src/components/enterprise-page/data/hooks.js

```diff
@@ -0,0 +1,65 @@
+import { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import camelcaseKeys from 'camelcase-keys';
+import { logError } from '@edx/frontend-platform/logging';
+
+import { fetchEntepriseCustomerConfig } from './service';
+
+const initialConfig = {
+  name: undefined,
+  slug: undefined,
+  uuid: undefined,
+  contactEmail: undefined,
+  branding: {
+    logoUrl: undefined,
+    banner: {
+      borderColor: '#007D88',
+      backgroundColor: '#D7E3FC',
+    },
+  },
+};
+
+// eslint-disable-next-line import/prefer-default-export
+export function useEnterpriseCustomerConfig() {
+  const { enterpriseSlug } = useParams();
+  const [enterpriseConfig, setEnterpriseConfig] = useState(initialConfig);
+
+  useEffect(() => {
+    fetchEntepriseCustomerConfig(enterpriseSlug)
+      .then((response) => {
+        const { results } = camelcaseKeys(response.data, { deep: true });
```

Comment:
Should we use `camelCaseObject` from `frontend-platform` here?

Reply:
Ah, yes. I forgot that frontend-platform provides a util function for this. I will update, thanks!

Reply:
Updated.